source/Mlos.Python/mlos/Optimizers/unit_tests/TestBayesianOptimizer.py

```diff
@@ -167,7 +193,14 @@ def test_bayesian_optimizer_on_simple_2d_quadratic_function_cold_start(self):
 
             bayesian_optimizer.register(input_values_df, target_values_df)
             if i > optimizer_config.min_samples_required_for_guided_design_of_experiments and i % 10 == 1:
-                print(f"[{i}/{num_iterations}] Optimum: {bayesian_optimizer.optimum()}")
+                _, all_targets = bayesian_optimizer.get_all_observations()
+                best_config, optimum = bayesian_optimizer.optimum()
+                print(f"[{i}/{num_iterations}] Optimum: {optimum}")
+                assert optimum.y == all_targets.min()[0]
+                assert input_space.contains_point(best_config)

```

Comment:
Just for the future, the __contains__ operator is overriden, so we can do:
```
best_config in input_space
```

Reply:
Thanks, that's cool!
Also, for one-line fixes, github allows you to suggest so I can immediately commit your suggestion :)

Do you want me to change this here and/or happy to approve otherwise?